practice/validator/site.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+// 1. Input
+/**
+ * Gathers what has been typed in the 'name' field for each event.
+ */
+function getUserName() {
+  return $('#name').val();
+}
+/**
+ * Gathers what has been typed in the 'DOB' field for each event.
+ */
+function getUserDOB() {
+  return $('#dob').val();
+}
+/**
+ * Gathers what has been typed in the 'Phone' field for each event.
+ */
+function getUserPhone() {
+  return $('#phone').val();
+}
+
+// 2. Transform Functions
+// No transform.
+
+// 3. Create
+var NAME_MATCH = /\w+\s\w+/;
+var DOB_MATCH = /\d\d\d\d-\d\d-\d\d/;
+var PHONE_MATCH = /\d\d\d-\d\d\d-\d\d\d\d/;
+/**
```

Comment:
Similar to above.  To get even fancier, since \d\d\d- is repeated twice, this can be grouped by parentheses and repeated.  Something like (\d{3}-){2}